.eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
run the format:fix command here. Shouldn't you have upload more configs? For example, have the eslint package installed in dev dependency and then start the config? 
Also, should be interesting to have a script command to automatically fix some errors